### ISSUE TITLE
Installation errors on CentOS Stream 9 (5.14.0-289) and RHEL 9 (5.14.0-162)

### ISSUE THREAD

**Issue by stannous** [NVIDIA/gdrcopy#255]
Hello,
Not sure if this is related to bug  #254, but we're having trouble installing
gdrcopy-kmod-2.3-1dkms.noarch (no errors with gdrcopy-2.3-1.x86_64)
on both CentOS Stream 9 (5.14.0-289) and RHEL9 (5.14.0-162).

```
]$ sudo dnf install gdrcopy-kmod
Last metadata expiration check: 0:24:22 ago on Mon 08 May 2023 08:05:38 AM PDT.
Dependencies resolved.
=====================================================================================================================================================
 Package                          Architecture               Version                          Repository                                        Size
=====================================================================================================================================================
Installing:
 gdrcopy-kmod                     noarch                     2.3-1dkms                        nvidia-dgx-custom-os-9-local                      30 k

Transaction Summary
=====================================================================================================================================================
Install  1 Package

Total size: 30 k
Installed size: 50 k
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                             1/1 
  Installing       : gdrcopy-kmod-2.3-1dkms.noarch                                                                                               1/1 
  Running scriptlet: gdrcopy-kmod-2.3-1dkms.noarch                                                                                               1/1 
Start gdrcopy-kmod installation.
Building and installing to all available kernels.
**This process may take a few minutes ...
modprobe: FATAL: Module gdrdrv not found in directory /lib/modules/5.14.0-289.el9.x86_64
/var/tmp/rpm-tmp.SiNR3t: line 37: /sbin/chkconfig: No such file or directory
Checking required modules: 
modinfo: ERROR: Module gdrdrv not found.
Module gdrdrv does not exist**


  Verifying        : gdrcopy-kmod-2.3-1dkms.noarch                                                                                               1/1 

Installed:
  gdrcopy-kmod-2.3-1dkms.noarch                                                                                                                      

Complete!
```
The package seems to install ok, but there does not appear to be a DKMS module created.

Thanks

**Comment by stannous** [comment]
Seems like subunit and subunit-devel are not available for RHEL9/CentOS Stream 9.

```
[lab@colossus-01 ~]$ sudo yum whatprovides /sbin/chkconfig
Updating Subscription Management repositories.
Last metadata expiration check: 4:03:40 ago on Mon 08 May 2023 09:17:27 AM PDT.
chkconfig-1.20-2.el9.x86_64 : A system tool for maintaining the /etc/rc*.d hierarchy
Repo        : rhel-9-for-x86_64-baseos-rpms
Matched from:
Provide    : /sbin/chkconfig

[lab@colossus-01 ~]$ sudo yum install dkms check check-devel subunit subunit-devel
Updating Subscription Management repositories.
Last metadata expiration check: 4:03:45 ago on Mon 08 May 2023 09:17:27 AM PDT.
Package dkms-3.0.3-1.el9.noarch is already installed.
Package check-0.15.2-6.el9.x86_64 is already installed.
Package check-devel-0.15.2-6.el9.x86_64 is already installed.
No match for argument: subunit
No match for argument: subunit-devel
Error: Unable to find a match: subunit subunit-devel
[lab@colossus-01 ~]$ 
```

**Comment by stannous** [comment]
This appears to have been fixed a while ago with  14bfff1886eefdd6c1adc777a0fb2e3ed0a7da33.

Thanks @pakmarkthub !
